tailwind.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
       vs: "348px",
       ...defaultTheme.screens,
       "3xl": "1920px",
+      print: { raw: "print" },
```

Comment:
what difference does this make? This is not required as of current tailwind version right, as tailwind already includes this modifier and given that it was working in existing usages so far.

Reply:
is it okay ? now